neworigins/extra.cpp

```diff
@@ -0,0 +1,1481 @@
+	EnableItem(I_MCROSSBOW);
+	EnableItem(I_MWAGON);
+	EnableItem(I_GLIDER);
+	EnableItem(I_NET);
+	EnableItem(I_LASSO);
+	EnableItem(I_BAG);
+	EnableItem(I_SPINNING);
+	EnableItem(I_LEATHERARMOR);
+	ModifyArmorFlags("LARM", ArmorType::USEINASSASSINATE);
+	ModifyWeaponAttack("DBOW",
+			ARMORPIERCING,
+			ATTACK_RANGED,
+			WeaponType::NUM_ATTACKS_HALF_SKILL);
+	// Make DBOWs require just LBOW, not XBOW?  And downgrade
+	// them from ARMORPIERCING to just PIERCING?
+	ModifyWeaponAttack("RUNE",
+			SLASHING,
+			ATTACK_COMBAT,
+			WeaponType::NUM_ATTACKS_HALF_SKILL);
+	EnableItem(I_CLOTHARMOR);
+	EnableItem(I_BOOTS);
+	EnableItem(I_BAXE);
+	// EnableItem(I_MBAXE);
+	// EnableItem(I_IMARM);
+	EnableItem(I_SUPERBOW);
+	EnableItem(I_LANCE);
+	EnableItem(I_JAVELIN);
+	EnableItem(I_PIKE);
+	EnableItem(I_MSHIELD);
+	EnableItem(I_ISHIELD);
+	EnableItem(I_WSHIELD);
+	EnableItem(I_AEGIS);
+	EnableItem(I_WINDCHIME);
+	EnableItem(I_GATE_CRYSTAL);
+	EnableItem(I_STAFFOFH);
+	EnableItem(I_SCRYINGORB);
+	EnableItem(I_CORNUCOPIA);
+	EnableItem(I_BOOKOFEXORCISM);
+	EnableItem(I_HOLYSYMBOL);
+	EnableItem(I_CENSER);
+	EnableItem(I_RELICOFGRACE);
+
+	// Cut down the number of trade items to improve
+	// chances of good trade routes
+	DisableItem(I_FIGURINES);
+	DisableItem(I_TAROTCARDS);
+	DisableItem(I_CAVIAR);
+	DisableItem(I_CHOCOLATE);
+	DisableItem(I_ROSES);
+	DisableItem(I_VELVET);
+	DisableItem(I_CASHMERE);
+	DisableItem(I_WOOL);
+
+	// EnableItem(I_FOOD);
+	// EnableSkill(S_COOKING);
+	// EnableSkill(S_CREATE_FOOD);
+
+	DisableItem(I_STAFFOFL);
+	DisableItem(I_GNOME);
+
+	// EnableSkill(S_ARMORCRAFT);
+	// EnableSkill(S_WEAPONCRAFT);
+	EnableSkill(S_ENCHANT_SHIELDS);
+	EnableSkill(S_CREATE_AEGIS);
+	EnableSkill(S_CREATE_WINDCHIME);
+	EnableSkill(S_CREATE_GATE_CRYSTAL);
+	EnableSkill(S_CREATE_STAFF_OF_HEALING);
+	EnableSkill(S_CREATE_SCRYING_ORB);
+	EnableSkill(S_CREATE_CORNUCOPIA);
+	EnableSkill(S_CREATE_BOOK_OF_EXORCISM);
+	EnableSkill(S_CREATE_HOLY_SYMBOL);
+	EnableSkill(S_CREATE_CENSER);
+	EnableSkill(S_TRANSMUTATION);
+	EnableSkill(S_BLASPHEMOUS_RITUAL);
+	EnableSkill(S_ENDURANCE);
+
+	ModifySkillDependancy(S_RAISE_UNDEAD, 0, "SUSK", 3);
+	ModifySkillDependancy(S_SUMMON_LICH, 0, "RAIS", 3);
+	// ModifySkillDependancy(S_CREATE_AURA_OF_FEAR, 0, "DEMO", 1);
+
+	ModifyItemMagicOutput(I_SKELETON, 200);
+	ModifyItemMagicOutput(I_WOLF, 200);
+	ModifyItemMagicOutput(I_IMP, 200);
+	ModifyItemMagicOutput(I_UNDEAD, 100);
+	ModifyItemMagicOutput(I_EAGLE, 100);
+	ModifyItemMagicOutput(I_DEMON, 100);
+	ModifyItemMagicOutput(I_DRAGON, 20);
+	ModifyItemMagicOutput(I_LICH, 20);
+	ModifyItemEscape(I_IMP,
+			ItemType::ESC_LEV_LINEAR | ItemType::LOSE_LINKED,
+			"SUIM",
+			20);
+	ModifyItemEscape(I_DEMON,
+			ItemType::ESC_LEV_LINEAR | ItemType::LOSE_LINKED,
+			"SUDE",
+			20);
+	ModifyItemEscape(I_BALROG,
+			ItemType::ESC_LEV_LINEAR | ItemType::LOSE_LINKED,
+			"SUBA",
+			20);
+	ModifyMonsterSpecial("LICH", "icebreath", 4);
+	ModifyMonsterSpecial("BALR", "hellfire", 4);
```

Comment:
@sgb based on your advice i changed it to hellfire 4. Does it sound balanced?

Reply:
I think it reasonable, but then I suggested it, so I would!